index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+
+<html lang="en">
+
+<head>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" type="text/css"
+        href="https://cdnjs.cloudflare.com/ajax/libs/normalize/8.0.1/normalize.min.css">
+    <link rel="stylesheet" type="text/css" href="styles.css">
+    <script type="text/javascript" src="src/globals.js" defer></script>
+    <script type="text/javascript" src="src/textSanitisation.js" defer></script>
+    <script type="text/javascript" src="src/punctuationIterator.js" defer></script>
+    <script type="text/javascript" src="src/wordSpanContents.js" defer></script>
+    <script type="text/javascript" src="src/fetchedTextProcessing.js" defer></script>
+    <script type="text/javascript" src="src/eventHandling.js" defer></script>
+    <script type="text/javascript" src="src/timer.js" defer></script>
```

Comment:
this file doesn't exist, I think it's a good idea to put timer stuff there though

Reply:
done